server/routers/share-recovery/recoverWallet.ts

```diff
@@ -12,28 +12,36 @@ import { BackupUtils } from "@/server/utils/backup/backup.utils";
 import { Config } from "@/server/utils/config/config.constants";
 import { getShareHashValidator } from "@/server/utils/share/share.validators";
 import { DbWallet } from "@/prisma/types/types";
+import { createHash } from "crypto";
 
 export const RecoverWalletSchema = z
   .object({
     walletId: z.string().uuid(),
     recoveryBackupShareHash: getShareHashValidator().optional(),
+    recoveryBackupShare: z.string().optional(),
```

Comment:
This cannot be sent to the server, as it implies we now have access to the wallet private key.

Reply:
fixed here: https://github.com/wanderwallet/embed-api/pull/25/commits/7fd4e20acb77226d327e2347929a7db7a88b9243